### PR TITLE
TRD fix halfchamberside and reduce err,warn messages

### PR DESCRIPTION
**Pull request by bazinski**
- fix halfchamberside and a/c side mix up for stats.
- reduce messages for config events, was 20 but then 20 per epn... now zero.
- add tracklet detail printout to headerverbose.

### CODE REVIEW

File: Detectors/TRD/reconstruction/src/DigitsParser.cxx [hunk 1]
```diff
@@ -51,8 +51,8 @@ int DigitsParser::Parse(std::array<uint32_t, o2::trd::constants::HBFBUFFERMAX>*
   mDetector = detector;
   mStack = stack;
   mLayer = layer;
-  mSide = side;
-  mHalfSM = detector * 2 + mSide;
+  mHalfChamberSide = side;
+  mHalfSM = detector * 2 + mHalfChamberSide;
```
Comment (martenole) [inline Detectors/TRD/reconstruction/src/DigitsParser.cxx]: What is `mHalfSM`? Seems it is not used anywhere and can be removed, right?

File: Detectors/TRD/reconstruction/src/TrackletsParser.cxx [hunk 2]
```diff
@@ -66,6 +66,15 @@ int TrackletsParser::Parse(std::array<uint32_t, o2::trd::constants::HBFBUFFERMAX
...
   mEventRecords = eventrecords;
+  //A HACK !!!
+  /*if(mFEEID.supermodule>15){
+    mHeaderVerbose=true;
+    LOG(info) << "Verbose output as we have detector : " << detector;
+  }
+  else {
+    mHeaderVerbose=options[TRDHeaderVerboseBit];
+  }*/
+  //END HACK !!!
```
Comment (martenole) [inline Detectors/TRD/reconstruction/src/TrackletsParser.cxx]: Is the hack supposed to stay in?

File: Detectors/TRD/reconstruction/include/TRDReconstruction/TrackletsParser.h [hunk 3]
```diff
@@ -78,11 +78,11 @@ class TrackletsParser
   void incParsingError(int error)
   {
     if (mOptions[TRDGenerateStats]) {
-      mEventRecords->incParsingError(error, mFEEID.supermodule, mFEEID.side, mStack * constants::NLAYER + mLayer);
+      mEventRecords->incParsingError(error, mFEEID.supermodule, mRobSide, mStack * constants::NLAYER + mLayer);
```
Comment (martenole) [inline Detectors/TRD/reconstruction/include/TRDReconstruction/TrackletsParser.h]: `mRobSide` is the same as `mHalfChamberSide` in DigitsParser.h right? Maybe it makes sense to give them the same name?

**Review by martenole — DISMISSED** [review]
Some minor comments, otherwise looks OK to me

File: Detectors/TRD/reconstruction/src/TrackletsParser.cxx (hunk 2 above)
Comment (bazinski) [reply Detectors/TRD/reconstruction/src/TrackletsParser.cxx]: ja its for debugging the last 2 supermodules, but i can take it out if it offends others.

File: Detectors/TRD/reconstruction/include/TRDReconstruction/TrackletsParser.h (hunk 3 above)
Comment (bazinski) [reply Detectors/TRD/reconstruction/include/TRDReconstruction/TrackletsParser.h]: Yes, it goes to what was already in the trackletparser class, its more sane in the new rawreader. will change

File: Detectors/TRD/reconstruction/src/DigitsParser.cxx (hunk 1 above)
Comment (bazinski) [reply Detectors/TRD/reconstruction/src/DigitsParser.cxx]: yes, was me testing why the stats were not coming out correct.

File: Detectors/TRD/reconstruction/src/TrackletsParser.cxx (hunk 2 above)
Comment (martenole) [reply Detectors/TRD/reconstruction/src/TrackletsParser.cxx]: no worries, just double-checking if that was a leftover or it is still used. Fine with me to keep it in for now

Comment (bazinski) [reply Detectors/TRD/reconstruction/src/TrackletsParser.cxx]: removed ;-)

File: Detectors/TRD/reconstruction/src/DigitsParser.cxx (hunk 1 above)
Comment (bazinski) [reply Detectors/TRD/reconstruction/src/DigitsParser.cxx]: mHalfSM removed
```

File: Detectors/TRD/reconstruction/include/TRDReconstruction/TrackletsParser.h (hunk 3 above)
Comment (bazinski) [reply Detectors/TRD/reconstruction/include/TRDReconstruction/TrackletsParser.h]: changed.

**Review by martenole — APPROVED** [review]

**Pull request merged**